README.md

```diff
@@ -15,12 +15,20 @@ To ensure the long-term health of the project, we are temporarily pausing all of
 
 ✅ We will continue reviewing and merging community PRs.
```

Comment:
Are you sure you want to do that? It does not make sense to me to add community features if you want to ensure a sustainable ecosystem.

Did you mean?

"We will continue reviewing and merging PRs with a focus on stability, security, cleanups, and debt reduction."

Reply:
But we are still reviewing PRs from the community. I believe we will continue to do so.

Reply:
I think it's worth deciding in the maintainer team. Adding new features means growing the complexity / maintenance scope.

Reply:
We already did :smile: - we had over 70 PRs already merged - mostly new features